pom.xml

```diff
@@ -545,6 +545,42 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>exec-maven-plugin</artifactId>
```

Comment:
Order `groupId` followed by `artifactId`, and add the version
```xml
<groupId>org.codehaus.mojo</groupId>
<artifactId>exec-maven-plugin</artifactId>
<version>3.0.0</version>
```

Reply:
Reordered and version added.